loader/src/main/java/net/neoforged/neoforgespi/Environment.java

```diff
@@ -14,9 +14,7 @@
 import java.util.function.Supplier;
 import net.neoforged.api.distmarker.Dist;
 import net.neoforged.neoforgespi.locating.IModDirectoryLocatorFactory;
-import net.neoforged.neoforgespi.locating.IModFile;
 import net.neoforged.neoforgespi.locating.IModLocator;
-import net.neoforged.neoforgespi.locating.ModFileFactory;
 
 /**
  * Global environment variables - allows discoverability with other systems without full forge
```

Comment:
we can probably delete this entire class - dist is already available in a bunch of other places (e.g. mod constructor injection, `FMLEnvironment.dist`)